Jenkinsfile.baguette

```diff
@@ -56,6 +63,26 @@ pipeline {
                         }
                     }
                 }
+                stage("License Scan") {
+                    environment {
+                        FOSSA_API_KEY=credentials('cb07b147-32a4-4400-aaac-21c3f8c9e62e')
+                    }
+                    agent {
+                        label 'ubuntu-1604-aufs-edge'
+                    }
+                    steps {
+                        withCredentials([hubCred]) {
```

Comment:
```
withCredentials([
    usernamePassword(credentialsId: 'orcaeng-hub.docker.com', usernameVariable: 'REGISTRY_USERNAME', passwordVariable: 'REGISTRY_PASSWORD'),
    string(credentialsId: 'fossa-api-key', variable: 'FOSSA_API_KEY')
]) { ... }
```

$class syntax is a legacy pipeline pre 1.0 compatibility syntax to be avoided as much as possible